<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDView</class>
 <widget class="QWidget" name="SDView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimDSP Run</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,1">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="outputLayout" stretch="3,0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="plotLayout" stretch="0,0">
       <item>
        <widget class="QGroupBox" name="inputBox">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Input (ADC) - Time</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="SDPlot" name="PlotA" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="outputBox">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Output (DAC) - Time</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="SDPlot" name="PlotB" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="inOutTime">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Input/Output (Time)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">timeFreqGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="inOutFreq">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Input/Output (Freq)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">timeFreqGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="inTimeFreq">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Input (Time/Freq)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">timeFreqGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="outTimeFreq">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Output (Time/Freq)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">timeFreqGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="appOutput">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="optionsLayout" stretch="0,0,0,0,0">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="fsLabel">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sampling Rate:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="fsValue">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>8000.0 Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="ch0Layout">
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="horizontalSpacing">
           <number>2</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="inputLabel">
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Input Signal:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="inputComboBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Sine</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Square</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sawtooth</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Noise</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>File</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Microphone</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="frequencyLabel">
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Frequency:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="frequencySpinBox">
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="suffix">
             <string>Hz</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>4000</number>
            </property>
            <property name="singleStep">
             <number>40</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="awgnCheckBox">
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>AWGN</string>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QGridLayout" name="snrLayout" columnstretch="3,0">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QSpinBox" name="snrSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>40</number>
              </property>
              <property name="value">
               <number>40</number>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="snrLabel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>Open Sans</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>SNR(dB)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="QDial" name="amplitudeDial">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="timeBaseLabel">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Time base (x10)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDial" name="timeBaseDial">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="accessibleName">
          <string/>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>true</bool>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="notchTarget">
          <double>3.700000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="amplitudeLabel">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Amplitude: 1V</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="SDKeyboard" name="widgetKeyboard" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SDKeyboard</class>
   <extends>QWidget</extends>
   <header>src/view/sdkeyboard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SDPlot</class>
   <extends>QWidget</extends>
   <header>src/view/sdplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="timeFreqGroup"/>
 </buttongroups>
</ui>
